core/src/hash_table/file/mod.rs

```diff
@@ -205,7 +213,18 @@ pub mod tests {
         let (table, _dir) = test_table();
 
         let re = |s, k| {
```

Comment:
I don't know what these closures are for. Could you rename them and also rename the params to something more expressive. + some docs would be nice

Reply:
I don't know either. I did not code this. It is not part of this PR.